app/controllers/films_controller.rb

```diff
@@ -1,16 +1,43 @@
 require 'pry'
+require 'open-uri'
 class FilmsController < ApplicationController
+  def add_movie
+    movie_id = params[:movie_id]
+      movie = Film.create(
+        rapid_id: params['movie_id'],
+        title: params['title'],
+        rating: params['rating'],
+        genre: params['genre'],
+        year: params['year']
+      )
+    if movie.save
+      if params['picture'].present?
+        movie.images.create(remote_picture_url: params['picture'])
+      redirect_to 'index.html.erb', notice: 'Movie added to the database.'
+
+    else
+      redirect_to 'index.html.erb', alert: 'Failed to add the movie to the database.'
+      end
+    end
+    end
 
   def index
-    # @movie_rows = YAML.load_file(Rails.root.join('config', 'movies.yml'))
-     @films = Film.all
+    @films = Film.all
+    movie_id = params[:movie_id]
+    #  @films = Film.where(title: movie_id)
+    response = HTTParty.get("https://movies-api14.p.rapidapi.com/search?query=#{movie_id}", headers: {
+      'X-RapidAPI-Host' => 'movies-api14.p.rapidapi.com',
+      'X-RapidAPI-Key' => '562c0c79a0msha9b8db4de63e59dp131a2djsna425bbbdbb01'
```

Comment:
use `.env` for these  X-RapidAPI-Key and X-RapidAPI-Host
https://github.com/bkeepers/dotenv

define it in the `.env` file 
TWISTO_URL=https://testcom

use it like this 
`ENV['TWISTO_URL']`